xAPI.md

```diff
@@ -2483,8 +2483,18 @@ See <a href="#AppendixE">Appendix E: Example Signed Statement</a> for an example
 
 ## 3.0 Metadata
 
+Metadata is additional information about the resource.  It enables decision making, search, and discoverability within 
+systems.  In xAPI, metadata can be utilized in a variety of locations.  The most common is within <a name="actdef"> Activity Definitions </a>, 
+where the metadata describes the Activity. Most types of metadata have the opportunity to have Extensions.  
+Other uses include Verbs, Activity types, and attachments.
+
 ### 3.1 IRI Requirements
 
+Up-to-date information is the most effective. xAPI metadata is no exception, which is why xAPI uses IRIs for identifiers.
+Using IRIs ensures uniqueness and promotes resolvability is the IRI.  The LRS and Activity each have responsibilities 
+in regard to each IRI as outline below.  Activity Definitions have additional rules which can be found in <a name="actdef"> 
+this section</a>.
+
 ###### Metadata Requirements
```

Comment:
Ok,  I must have thought I could originally moved this.  Now, after looking at the Verb IRIs expanding, thinking either a reference or moving it here will be better.  To not conflict with PRs, I'm going to make an Issue to move this section back where it belongs and empty it for now.  Once we have the generic IRI language done, we'll see if it makes more sense in Part 1 Section 8.3 or in here.  Either way this section will reference the metadata in the Activity Definitions.  Sorry for the mix-up in this and great catch!